src/ocean/util/log/model/ILogger.d

```diff
@@ -1,60 +1,93 @@
+    License:
+        Tango Dual License: 3-Clause BSD License / Academic Free License v3.0.
+        See LICENSE_TANGO.txt for details.
+
+    Version: Initial release: May 2004
+
+    Authors: Kris
 
 *******************************************************************************/
 
 module ocean.util.log.model.ILogger;
 
 import ocean.transition;
 
-/*******************************************************************************
-
-*******************************************************************************/
 
+/// Ditto
 interface ILogger
 {
-        enum Level {Trace=0, Info, Warn, Error, Fatal, None};
+    /// Defines the level at which a message can be logged
+    public enum Level
+    {
+        ///
+        Trace = 0,
+        ///
+        Info,
+        ///
+        Warn,
+        ///
+        Error,
+        ///
+        Fatal,
+        ///
+        None
+    };
 
 
-        /***********************************************************************
+    /***************************************************************************
 
-                Context for a hierarchy, used for customizing behaviour
-                of log hierarchies. You can use this to implement dynamic
-                log-levels, based upon filtering or some other mechanism
+        Context for a hierarchy, used for customizing behaviour of log
+        hierarchies. You can use this to implement dynamic log-levels,
+        based upon filtering or some other mechanism
 
-        ***********************************************************************/
+    ***************************************************************************/
 
-        interface Context
-        {
-                /// return a label for this context
-                istring label ();
+    public interface Context
+    {
+        /// return a label for this context
+        public istring label ();
 
-                /// first arg is the setting of the logger itself, and
-                /// the second arg is what kind of message we're being
-                /// asked to produce
-                bool enabled (Level setting, Level target);
-        }
+        /// first arg is the setting of the logger itself, and
+        /// the second arg is what kind of message we're being
+        /// asked to produce
+        public bool enabled (Level setting, Level target);
+    }
 
 
-        /***********************************************************************
+    /***************************************************************************
 
-                Is this logger enabed for the specified Level?
+        Returns:
+            `true` if this logger is enabed for the specified `Level`
 
-        ***********************************************************************/
+        Params:
+            `Level` to test for, defaults to `Level.Fatal`.
+
+    ***************************************************************************/
+
+    public bool enabled (Level level = Level.Fatal);
+
+
```

Comment:
You introduced two blank lines here where as the rest of the functions have a blank-line distance of one.4
Please keep it consistent :)